### PR TITLE
Add code to access remote `Case` entity

### PR DESCRIPTION
**Pull request by dontub**
The corresponding CiviCRM extension is here: https://github.com/systopia/remote_case

It's basically the same as in `civicremote_activity` adapted for cases.

systopia-reference: 25874

### CODE REVIEW

File: modules/civiremote_case/civiremote_case.routing.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    _controller: 'Drupal\civiremote_case\Controller\CaseUpdateController::form'
+  options:
+    no_cache: TRUE
+    parameters:
+      id:
+        type: int
+      profile:
+        type: string
+  requirements:
+    _user_is_logged_in: 'TRUE'
```
Comment (jensschuppe) [inline modules/civiremote_case/civiremote_case.routing.yml]: Is this always required?

Comment (dontub) [reply modules/civiremote_case/civiremote_case.routing.yml]: Actually I cannot imagine a use case where anyone should be able to edit a case without any authorization. If it'd actually be necessary a custom module with a custom route could be used.

Comment (dontub) [reply modules/civiremote_case/civiremote_case.routing.yml]: I think it make sense to add a new permission here. I'll change that.

Comment (dontub) [reply modules/civiremote_case/civiremote_case.routing.yml]: Access is now limited to permission `access civiremote case`.

**Review by jensschuppe — APPROVED** [review]

**Pull request merged**